src/cr_signal.cc

```diff
@@ -1,4 +1,5 @@
 #include <glog/logging.h>
+#include <cinttypes>
```

Comment:
Put it together with other STL headers and sort them alphabetically.

Reply:
fine